speech/google/cloud/speech/client.py

```diff
@@ -117,14 +117,10 @@ def async_recognize(self, sample, language_code=None,
         if sample.encoding is not Encoding.LINEAR16:
             raise ValueError('Only LINEAR16 encoding is supported by '
                              'asynchronous speech requests.')
-
-        data = _build_request_data(sample, language_code, max_alternatives,
-                                   profanity_filter, speech_context)
-
-        api_response = self.connection.api_request(
-            method='POST', path='speech:asyncrecognize', data=data)
-
-        return Operation.from_api_repr(self, api_response)
+        api = self.speech_api
+        response = api.async_recognize(sample, language_code, max_alternatives,
+                                       profanity_filter, speech_context)
+        return response
```

Comment:
Indeed why not?

Updated:
https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2624/commits/57bb55747593b5d2c68f46d2edef87d2bcf45f77